README.md

```diff
@@ -83,3 +175,7 @@ in the process.
 * Thanks to [Keplr](https://www.welcometothejungle.com/fr/companies/keplr) for allowing me to contribute to this side-project of mine during paid work hours.
 
 ![](sponsors/jetbrains.svg?display=inline-block) ![](sponsors/keplr.png?display=inline-block)
```

Comment:
Haven't been for years now.
```suggestion
```

Reply:
Shall I do this removal in a separate PR?

Reply:
No preference on my side, resolving.